plumbing/cache/object_test.go

```diff
@@ -67,6 +69,20 @@ func (s *ObjectSuite) TestClear(c *C) {
 	c.Assert(obj, IsNil)
 }
 
+func (s *ObjectSuite) TestConcurrentAccess(c *C) {
+	var wg sync.WaitGroup
+
+	for i := 0; i < 1000; i++ {
+		wg.Add(1)
+		go func(i int) {
+			s.c.Put(newObject(fmt.Sprint(i), FileSize(i)))
```

Comment:
Do we also want to exercise the other operations that require synchronization? Like Get and Clear?